solidity/test/rewards/TestRewardUpgrades.js

```diff
@@ -168,6 +168,60 @@ describe("Rewards/Upgrades", () => {
             )
         })
 
+        it("allocates interval before finalize rewards upgrade", async () => {
+            await time.increase(termLength + 1) // interval 0 ends
+
+            await rewards.initiateRewardsUpgrade(
+                newRewards.address,
+                {from: owner}
+            )
+
+            await time.increase(termLength + 1) // interval 1 ends
+            await rewards.setCloseTime(timestamps[2])
+
+            await rewards.allocateRewards(1)
+            await rewards.finalizeRewardsUpgrade({from: owner})
+
+            const upgradeInitiatedTimestamp = await rewards.upgradeInitiatedTimestamp()
+            const upgradeFinalizedTimestamp = await rewards.upgradeFinalizedTimestamp()
+
+            expect(upgradeInitiatedTimestamp).to.eq.BN(0)
+            expect(upgradeFinalizedTimestamp).not.to.eq.BN(0)
+        })
+
+        it("transfers amount to a new contract upon rewards upgrade", async () => {
```

Comment:
This test makes a lot of sense but the test name does not express what we are really testing here and at the first sight I was sure it's duplicating `""moves all unallocated rewards to new contract"` test case.

What do you think about renaming this test to e.g. `transfers any topped-up amount to a new contract after finalizing the upgrade`?

Reply:
Yeah, sounds good, changed.